src/RA_Log.h

```diff
@@ -2,9 +2,41 @@
 #define RA_LOG_H
 #pragma once
 
-extern void RADebugLogNoFormat(const char* data);
-extern void RADebugLog(const char* sFormat, ...);
+#include "services\ILogger.hh"
 
-#define RA_LOG RADebugLog
+#ifdef RA_UTEST
+
+#define RA_LOG_LEVEL(lvl, ...)
+#define RA_LOG(...)
+
+#else
+
+#include "RA_StringUtils.h"
+
+#include "services\ServiceLocator.hh"
+
+#define RA_LOG_LEVEL(lvl, ...) \
```

Comment:
Not using macros makes it harder to undefine these functions for the unit tests. However, it would be easy enough to provide a MockLogger or something similar to swallow the messages.